src/browser/cacheInput.js

```diff
@@ -169,6 +169,11 @@ function CacheInputController($scope, simDriver, fileParser) {
         ctrl.cacheInfo = data;
     });
 
+    ctrl.runSimulation = function() {
+        let val = ipcRenderer.send('runSimulation', ctrl.cacheInfo);
```

Comment:
since this is not synchronous there won't be a return value so all you'll need is ` ipcRenderer.send('runSimulation', ctrl.cacheInfo);`